spec/features/new_project_spec.rb

```diff
@@ -190,6 +190,11 @@
     )
   end
 
+  it "does not make invalid config because of bullet config in development" do
+    expect(development_config).to match(/^ +Bullet.enable = true$/)
+    expect(development_config).to_not match(/trueconfig/)
```

Comment:
Could we add this expectation into the spec below rather than introducing a new one? Normally I would prefer it as a separate spec the way you have done it, but these tests are really slow because they run the whole suspenders command as setup.

Reply:
ok, i merged the two tests